pyanaconda/dbus_addons/baz/baz.py

```diff
@@ -0,0 +1,39 @@
+# bar_module.py
```

Comment:
Bad name and I don't think it should be here at all.

Reply:
Used correct name & we should address this globally in a follow up PR.